testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -612,20 +617,28 @@ impl BitcoinRegtestController {
     }
 
     fn prepare_tx(
-        &self,
+        &mut self,
         public_key: &Secp256k1PublicKey,
         ops_fee: u64,
+        attempt: u64,
     ) -> Option<(Transaction, Vec<UTXO>)> {
         let tx_fee = self.config.burnchain.burnchain_op_tx_fee;
         let amount_required = tx_fee + ops_fee;
 
-        // Fetch some UTXOs
-        let utxos = match self.get_utxos(&public_key, amount_required) {
-            Some(utxos) => utxos,
-            None => {
-                debug!("No UTXOs for {}", &public_key.to_hex());
-                return None;
-            }
+        let utxos = if attempt > 1 && self.last_utxos.len() > 0 {
+            // in RBF, you have to consume the same UTXOs
```

Comment:
so when picking an UTXO, it probably should chose one with enough for several attempts

Reply:
Yeah, UTXO selection is pretty dumb right now.  I'll make it so it greedily builds a transaction from UTXOs in decreasing value, which should both reduce the number of UTXOs consumed per transaction as well as ensure that there's an adequate buffer for RBF.